front/bill.form.php

```diff
@@ -73,7 +73,7 @@
                         $ic->update($fields);
                      }
                   }
-                  
+
```

Comment:
Heu.. Ok for this one, but I'm not going to delete all extra lines/comments/whatever on all plugins I've worked on :D